library/ui-styles/src/main/res/values/palette.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="UnusedResources">
+
+    <!-- Ignore the lint error UnusedResources for all the resources in this file -->
```

Comment:
Is there a point to this comment? I don't see the error anywhere in the file

Reply:
You do not see the error since I have added `tools:ignore="UnusedResources"` above :)

Reply:
Ahh in this case do we need the comments? They just repeat what the code already describes well (sorry if I'm a bit pedantic about these things 😂✌️)

Reply:
You are right. I will add some justification to make the comment a bit more valuable

Reply:
0122d8f9330ea1db5cd9c8d95763c6ad0af04662